plugins/api/src/atomic_apply_metadata_operations.cpp

```diff
@@ -523,8 +523,8 @@ auto plugin_factory(const std::string& _instance_name,
                         RODS_API_VERSION,                           // API version
                         NO_USER_AUTH,                               // Client auth
                         NO_USER_AUTH,                               // Proxy auth
-                        "BytesBuf_PI", 0,                           // In PI / bs flag
-                        "BytesBuf_PI", 0,                           // Out PI / bs flag
+                        "BinBytesBuf_PI", 0,                           // In PI / bs flag
```

Comment:
line up the comments

Reply:
now done on both branches, correcting comment alignment